environment.yml

```
@@ -12,6 +12,7 @@ dependencies:
   - pip
   - pyyaml
   - copernicusmarine >= 2
+  - openpyxl >= 3.1.5
```

```suggestion
  - openpyxl
```
I don't think we need to specify the lower bound. Its best to be flexible unless there is an incompatibility with x version in our codebase